sdk/cosmos/cosmos-emulator-matrix.json

```
@@ -0,0 +1,60 @@
+{
+    "displayNames": {
+      "--disablecov": "",
+      "false": "",
+      "true": ""
+    },
+    "matrix": {
+      "Agent": {
+        "windows-2022": { "OSVmImage": "MMS2022", "Pool": "azsdk-pool-mms-win-2022-general" }
+      },
+      "EmulatorConfig": {
+        "Emulator Tests Python 3.8": {
+            "PythonVersion": 3.8,
+            "CoverageArg": "--disablecov",
+            "TestSamples": "false",
+            "ToxTestEnv": "'whl,sdist'",
+            "ACCOUNT_HOST": "https://localhost:8081/",
+            "ACCOUNT_KEY": "C2y6yDjf5/R+ob0N8A7Cgv30VRDJIWEHLM+4QDU5DE2nQ9nDuVTqobD4b8mGGyPMbIZnqyMsEcaGQy67XIw/Jw==",
+            "Skip.Analyze": "true"
+        },
+        "Emulator Tests Python 3.9": {
+            "PythonVersion": 3.9,
+            "CoverageArg": "--disablecov",
+            "TestSamples": "false",
+            "ToxTestEnv": "'whl,sdist'",
+            "ACCOUNT_HOST": "https://localhost:8081/",
+            "ACCOUNT_KEY": "C2y6yDjf5/R+ob0N8A7Cgv30VRDJIWEHLM+4QDU5DE2nQ9nDuVTqobD4b8mGGyPMbIZnqyMsEcaGQy67XIw/Jw==",
+            "Skip.Analyze": "true"
+        },
+        "Emulator Tests Python 3.10": {
+            "PythonVersion": 3.10,
```

```suggestion
            "PythonVersion": "3.10",
```

Somewhere along the line the `0` is getting stripped. We've only ever tested this against string values for python version, given that `3.10.0` (if you specified patch version) isn't a number that is parsable. A lot of effort to auto-cast when we could just surround in quotes 👍